### DISCUSSION TITLE
LaTeX-incompatible input and strict mode is set to 'warn'

### DISCUSSION THREAD

**Discussion by OatLatte** [remarkjs/remark#1005] [Q&A] [answered]
I am using MDX in my NEXT.JS project with the react-math and rehype-katex plusgins. When I rendering the MDX files it gives me erros:

"LaTeX-incompatible input and strict mode is set to 'warn': In LaTeX, \ or \newline does nothing in display mode [newLineInDisplayMode]"

Flowing are my next.config.mjs file:

```
import nextMDX from '@next/mdx';
import remarkMath from 'remark-math';
import rehypeKatex from 'rehype-katex';

const nextConfig = {
  reactStrictMode: true,
};

const withMDX = nextMDX({
  extension: /\.mdx?$/,
  options: {
    remarkPlugins: [remarkMath],
    rehypePlugins: [rehypeKatex],
    // If you use `MDXProvider`, uncomment the following line.
    providerImportSource: '@mdx-js/react',
  },
});

export default withMDX({
  // Append the default value with md extensions
  nextConfig,
  pageExtensions: ['js', 'jsx', 'md', 'mdx'],
});
```

Any one knows how can I fix this or turn off the strict mode warnings? Thanks !

**Comment by wooorm** [comment, marked as answer]
- `rehype-katex` has options.
- There is no MDX code here, so I can’t help you with problems in your MDX code

**Reply by OatLatte** [reply]
Hello Wooorm,
Thank you very much for the reply! I just saw it. I am still having this problem, I just attached the MDX code here.

`
$$
\frac{d y}{d x}=\frac{d y}{d u} \times \frac{d u}{d x}\\[1em]
\text{If }y=(f(x))^n\text{ , then }\frac{dy}{dx}=n(f(x))^{n-1}f^{\prime}(x)\\[1em]
\text {If } y=f(g(x)) \text { , then } \frac{d y}{d x}=f^{\prime}(g(x)) g^{\prime}(x)
$$
`

**Reply by OatLatte** [reply]
![image](https://user-images.githubusercontent.com/103306071/179397125-37d703c0-d322-47b1-824d-01c88251e8bb.png)
Oh! GitHub auto-converted my TeX into formulas, I uploaded a new screenshot here.

**Reply by wooorm** [reply]
Can you please follow my previous suggestions on the fact that there are options for `rehype-katex`?
Here they are: https://github.com/remarkjs/remark-math/tree/main/packages/rehype-katex#unifieduserehypekatex-options.
Clicking through, and inferring what you say, it seems like you want to pass `{strict: false}`.
Finally, if you are unsure how to pass options, see our docs: https://mdxjs.com/docs/extending-mdx/#using-plugins.
